script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -556,10 +580,10 @@ Her face became serious with anger like I've never seen before.
 [sha:52e47edef494d08a0ee5374a0c83a7f8233ae4c9]{
 -- Page 115, Offset 30864.
 -- ……………………いいかげん、
-.........Besides,
+.........It was actually,
```

Comment:
I prefer besides just from a readability standpoint

Reply:
The reason I tweaked this is because the line above is already a bit of a fluff word - 
> Well, whatever. 
> Besides,

just read like too many dithering words in a row